src/util/debug_module.js

```diff
@@ -41,7 +41,7 @@ util.inspect.defaultOptions.depth = 10;
 util.inspect.defaultOptions.colors = true;
 util.inspect.defaultOptions.breakLength = Infinity;
 
-
+process.env.LOG_TO_SYSLOG_ENABLED = config.LOG_TO_SYSLOG_ENABLED ? 'true': 'false';
```

Comment:
this should be inside config.js like we do for GPFS_DL_PATH

Reply:
@romayalon In that case user should add the `LOG_TO_SYSLOG_ENABLED` and `LOG_TO_STDERR_ENABLED` to config.js. Otherwise `process.env` wont have these values.